src/docs/sphinx/dev_guide/tensor_dual.rst

```diff
@@ -0,0 +1,405 @@
+.. _header-n0:
+
+============
```

Comment:
Since this has Tensor and Dual class in it.. can you break this into a section that contains more manageable chunks? Also realize I haven't gotten through this yet...

Something like:

* Some common page (name tbd)
   - Tensor Class
   - Dual Class
   - Common utility accessors
   - Using them together!